### PR TITLE
Log in (UI)

### PR DESCRIPTION
**Pull request by Ernest103**
Created Three new pages:
LobbyPage
LogInForm
SignUpForm

### CODE REVIEW

**Review by Samuelmm97 — APPROVED** [review]
LGTM, I would wait for @brandongevat for a final review

File: tip_tracker/lib/core/auth/login/LobbyPage.dart
Comment (Samuelmm97) [inline tip_tracker/lib/core/auth/login/LobbyPage.dart:48]: For these hex codes I would assign them to some variable describing what this color is just for more clarity.

File: tip_tracker/lib/core/auth/login/LobbyPage.dart
Comment (Samuelmm97) [inline tip_tracker/lib/core/auth/login/LobbyPage.dart:18]: not sure if this is a flutter thing, but do these have to be named child, or can they be named something more descriptive?

Comment (brandongevat) [reply tip_tracker/lib/core/auth/login/LobbyPage.dart:18]: it does have to be child. pretty much every widget is just a parent of another widget.

File: tip_tracker/lib/core/auth/login/LobbyPage.dart
Comment (brandongevat) [reply tip_tracker/lib/core/auth/login/LobbyPage.dart:48]: you can create a new file under lib/config/themes named something like colors.dart, create a class Colors and start creating a bunch of static variables for each color you want to use.

**Pull request merged**
